### ISSUE TITLE
Pass values as props between views

### ISSUE THREAD

**Issue by EzeInverni** [joaquindk/react-native-tinder-navigator#2]
Hi! I implemented this navigator to load three imported views and I want to know how to pass some values as props when I touch on the the tab button to change between them. If it helps, i'm also using react-native-router-flux in my project. Thanks!

**Comment by joaquindk** [comment]
Thanks for you feedback @EzeInverni . With the current version of the navigator this is not possible unless you use a 3rd party lib like Redux to do it. However I'll try to incorporate it into the navigator asap and let you know when it's done.

**Comment by EzeInverni** [comment]
Hi, thanks for the reply! If anyone encounters the same problem, I 'solved' it by saving the values from a tab in AsyncStorage and then retrieving them in the other tab when componentWillReceiveProps() is called, wich happens everytime you press the buttons to swing between the tabs. A little bit too overkill but works for me.

**Comment by joaquindk** [comment]
Fair enough - I'll try to solve it using the new context api, which should provide a bit of a lighter solution. Will let you know when it's ready.
